php/php82/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM gcr.io/gcp-runtimes/ubuntu_20_0_4
 ENV PHP_DIR=/opt/php82
-ENV PHP_SRC_DIR=/usr/local/src/php80-build
+ENV PHP_SRC_DIR=/usr/local/src/php82-build
```

Comment:
Was this correct before?

Reply:
It's just the same of the env var (`PHP_SRC_DIR`) for the PHP source, so basically PHP 8.2 was being sourced in a directory called `php80`. It's not a functional problem, but I didn't like it, so I changed it.